src/autogluon/cloud/predictor/cloud_predictor.py

```diff
@@ -556,6 +556,7 @@ def predict(
         custom_image_uri: Optional[str] = None,
         wait: bool = True,
         backend_kwargs: Optional[Dict] = None,
+        **kwargs,
```

Comment:
add docstring

Reply:
Add example code should be added to tutorials that showcase specifying kwargs. Otherwise it will be hard for users to realize how to do this.

Reply:
Good idea. I will add some tutorials with this PR.